### ISSUE TITLE
DataTable: `virtualScroller` height not set on initial mount

### ISSUE THREAD

**Issue by makieno** [primefaces/primereact#1481]
### There is no guarantee in receiving an immediate response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeReact PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Codesandbox Case (Bug Reports)**
Please fork the codesandbox below and create a case demonstrating your bug report. Issues without a codesandbox have much less possibility to be reviewed.

https://codesandbox.io/s/primereact-test-5ugkx

**Current behavior**
Table body is not visible

**Expected behavior**
Table body should be visible

**Minimal reproduction of the problem with instructions**
When you provide a `virtualScroll` `DataTable` with all the necessary props in its initial render to render the first few elements, the `body` of the table is not visible, because the `virtualScroller` has no height set.
The source of the bug can be [found here](https://github.com/primefaces/primereact/blob/dbef5c82ea69df58968e56cfe0431ffb007566e4/src/components/datatable/ScrollableView.js#L70). The `height` is only set on update but not on mount.

* **React version:**
16.8

* **PrimeReact version:**
4.2.2

* **Browser:** 
all
 
* **Language:**
all

**Comment by fabiokiatkowski** [comment]
Same here.

**Comment by konradkoc** [comment]
Have the same issue with Prime 7.0.0, have to trigger hard DataTable rerender and then it works but only sometimes

**Comment by melloware** [comment]
This is fixed in 8.0.0 see the sandbox here: https://codesandbox.io/s/primereact-test-forked-4eik7h?file=/src/index.js
